### PR TITLE
chore(Python): added catalan sequence

### PR DESCRIPTION
**Pull request by Vedant488**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Prints the first n terms of the Catalan sequence

**Other information:**

NA

### CODE REVIEW

**Comment by Vedant488** [comment]
@Arsenic-ATG, @atin, @sridhar-5, @cyberwizard1001

Prints the Catalan number sequence. Please review the code.

File: algorithms/Python/dynamic_programming/catalan_sequence.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Catalan number
+
+"""
+Output: 
+The first 10 terms of Catalan sequence are :
+1 1 2 5 14 42 132 429 1430 4862
+"""
+
+
+def catalan(n):
```
Comment (atin) [inline algorithms/Python/dynamic_programming/catalan_sequence.py:11]: Add doctests like the one in interval_scheduling.py, binary_search.py etc

Comment (Vedant488) [reply algorithms/Python/dynamic_programming/catalan_sequence.py:11]: I have added the doctests. Please check.

**Comment by ming-tsai** [comment]
Hi @atin,
Please check it out

**Review by Arsenic-ATG — APPROVED** [review]

**Comment by Vedant488** [comment]
Hello @ming-tsai as this pull request was approved by @Arsenic-ATG , I think the review from @atin is no longer required. Would it be possible to remove the pending review and merge my branch ?

**Comment by Arsenic-ATG** [comment]
@Vedant488 don't worry, ming-tsai will merge it sooner or later.

**Comment by Vedant488** [comment]
@Arsenic-ATG ok. thanks 👍:smile:

**Review by atin — APPROVED** [review]

**Comment by ming-tsai** [comment]
Hi @Vedant488,
Now you are invited

**Comment by ming-tsai** [comment]
Hi @atin,
Do not merge pull request

**Pull request merged**